xbmc/pvr/windows/GUIEPGGridContainerModel.cpp

```diff
@@ -325,7 +326,7 @@ void CGUIEPGGridContainerModel::FindChannelAndBlockIndex(int channelUid, unsigne
 
 unsigned int CGUIEPGGridContainerModel::GetGridStartPadding() const
 {
-  unsigned int iEpgLingerTime = g_advancedSettings.m_iEpgLingerTime;
+  unsigned int iEpgLingerTime = CServiceBroker::GetSettings().GetInt(CSettings::SETTING_EPG_PAST_DAYSTODISPLAY) * 24 * 60;
```

Comment:
could we please rename the variable to `iPastMinutesToDisplay` or something similar to the setting name.

Reply:
sure